projects/plugins/jetpack/extensions/blocks/dialogue/dialogue.php

```diff
@@ -36,210 +36,16 @@ function register_block() {
 }
 add_action( 'init', __NAMESPACE__ . '\register_block' );
 
```

Comment:
None of the PHP rendering code is needed any more, since we are outputting the full dialogue markup via the `save` function.

Reply:
It's a big change. From here, Dialogue blocks will be 100% independent of the Conversation block, which is good IMO, but will require keeping the data constantly synced. `Conversation` block will become just a temporary context-provider.